CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Fix a Rails startup bug when validating steps that use translations from locale
   files. (#211)
+- Fix a typo in message.html.erb.tt.
```

Comment:
Actually, this should go above, under the Unreleased section.

Reply:
Thanks, @subzero10! Fixed in 9531636814a046257e1599925b55d02499af65aa.

Reply:
Thank you for catching the typo!